### DISCUSSION TITLE
`Ctrl` + `Shift` + `V` (paste without formatting) does not work on Firefox

### DISCUSSION THREAD

**Discussion by Marcono1234** [community/community#14102] [Other Feature Feedback, Questions, & Ideas] [answered]
Recently, GitHub added a [feature to paste markdown links](https://github.blog/changelog/2022-02-02-paste-links-directly-in-markdown/). To paste as plain text the combination <kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>V</kbd> has to be used.
The issue is that this does not work in Firefox: [bug 968104](https://bugzilla.mozilla.org/show_bug.cgi?id=968104), [bug 1707205](https://bugzilla.mozilla.org/show_bug.cgi?id=1707205)

So while the introduction of this new GitHub feature would in theory be useful, it can actually become quite annoying on Firefox at the moment. Would it be possible to investigate whether GitHub could implement a workaround (maybe by listening for key events) in the meantime?

Firefox 98.0.2 (64-Bit) on Windows 10, in [Troubleshoot Mode](https://support.mozilla.org/en-US/kb/diagnose-firefox-issues-using-troubleshoot-mode) (all extensions disabled)

**Comment by moorepants** [comment]
Is there a way to disable this new feature? It is actually quite a frustrating change. If I want to copy a url I would do that with my browser's features. I now find myself having to delete urls and markdown formatting on way too many pastes because so many things are hyperlinks that you may copy.

**Reply by Marcono1234** [reply]
> I now find myself having to delete urls and markdown formatting on way too many pastes

Is this because you are also using Firefox (or a different browser?) where <kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>V</kbd> is not working?
Or did you mean this in general because you cannot familiarize yourself with <kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>V</kbd>?

**Comment by moorepants** [comment]
I do use Firefox.

Anyone can familiarize themselves with a new command. But the new command should be associated with the new feature, not the other way around.

**Comment by Marcono1234** [comment, marked as answer]
Looks like this was fixed in the latest Firefox versions, <kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>V</kbd> now works for me.
